docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -465,24 +465,41 @@ for changing an arc from a household in B to a new housing unit in C.
 
 I could imagine making these rates quite complex someday, to take into
 account the age, sex, race/ethnicity, household structure, and even
-past migration history.  But at this point, it is not clear how
-complex is necessary to have a successful synthetic data set for
-testing PRL algorithms, so we will keep it quite simple.
+past migration history.  At this point, it is clear that age is
+necessary to get the college dormatory migration right, so we might as
+well include sex and race/ethnicity stratification in the rates as
+well.
+
+A complex type of movement that we need to capture is moving into and
+out of Group Quarters; it is useful to think of six broad types of GQ
+for PRL purposes grouped into two categories: non-institutional
+(college, military, other non-institutional); and institutional
+(carceral, nursing homes, and other institutional).  College is likely
+to be the tough one in Census applications (Census will have SSN for
+most military and incarcerated, Medicare for most nursing home, but
+people living in dorms, especially who don't file their own tax
+returns might not be PIK-able.)
```

Comment:
what does PIK able mean?

Reply:
Sorry for the acronym soup.  PIK = Protected Identification Keys

A few relevant links: https://www.census.gov/content/dam/Census/about/about-the-bureau/adrm/data-linkage/events/2017-11/Access%20to%20Innovation.pdf
https://www.census.gov/content/dam/Census/library/working-papers/2014/adrm/carra-wp-2014-01.pdf